model/src/main/scala/za/co/absa/atum/model/dto/AdditionalDataSubmitDTO.scala

```diff
@@ -0,0 +1,6 @@
+package za.co.absa.atum.model.dto
+
+case class AdditionalDataSubmitDTO (
+  atumPartitioning: PartitioningDTO,
+  additionalData: Map[String, Option[String]]
```

Comment:
additional field required: `author`

Reply:
I already added it here: https://github.com/AbsaOSS/atum-service/pull/143 and updated the client's code